### PR TITLE
chore: Style updates and README

### PR DESCRIPTION
**Pull request by mhomolak**
Example of how to in readme
Transition type changed
appropriate spacing for first child
Capitalization in file names

### CODE REVIEW

**Comment by StommePoes** [comment]
Issues:
when trying to npm start the demo, I get an error "Module not found" because AlertList is looking for Alert.js but it's alert.js. I assume the file should be called Alert.js.

The roles are not going to read out anything to a screen reader. Alert roles need to be present on the page when it loads. When text is added or changed inside a box with an alert role, that changed content is read out.

But here, the elements with the role are appearing and disappearing.  In order for Alerts to truly be live regions, you'll have to put the role="alert" on the span called "alertList".

We have list item elements inside a span. List items must be inside a list element. 

The alerts have close buttons but then we have to decide: are we alerting people of something they need to deal with and so should be move the keyboard focus to the close button? Or is it an alert that appears on the page with an alert role where users must know they exist but may continue doing whatever they were doing on the page and focus does not move?

If focus should be moved and the user's business interrupted, then these should be alertdialogs rather than alerts. Alerts generally don't have any focusables inside them. Alertdialogs do (they are the HTML equiv of Javascript Confirm() or Alert() warnings).

**Comment by mhomolak** [comment]
#1 and #4 are handled now.  Looking into the others

**Comment by StommePoes** [comment]
Cool, I'll test the #4 +role change on some SRs today

**Comment by StommePoes** [comment]
Hm.

So the alert role is overriding the native semantics of the ul (and li, I see they still have alert roles). I should have thought of that initially.
I think we need
```
<div role="alert">
<ul class="alertlist">
... stuff li's in here...
</ul>
</div>
```

Since divs don't have any native roles to begin with. Although, another question is, will we actually have a list of alerts on any given page? Or is it simply that the component must be able to spit out whatever and however many alerts that are necessary for a page situation? 

I don't think it would be wrong to have
```
<div role="alert">
...added... <p class="pe-alert">Warning, Trump elected</p>
...
</div>
```

If we still need close buttons (instead of the alert being allowed to remain on the page or fading away on its own)

```
<div role="alert">
  ...added... <div class="pe-alert">Warning: Betsy DeVos has now dismantled the Department of Education and turned it into Department for Religious Learning
  <button type="button">close</button></div>
</div>
```

File: src/js/Icon.js [hunk 1]
```diff
@@ -4,9 +4,8 @@ const Icon = (props) => {
 
   return (
     <svg version = "1.1"
```
Comment (StommePoes) [inline src/js/Icon.js]: you don't need the version anymore either \o/

**Comment by StommePoes** [comment]
looks good.

**Pull request merged**